samples/python/rag/README.md

```diff
@@ -38,10 +38,24 @@ export_tokenizer(tokenizer, output_dir)
 
 Install [deployment-requirements.txt](../../deployment-requirements.txt) via `pip install -r ../../deployment-requirements.txt` and then, run a sample:
 
-`python text_embeddings.py BAAI/bge-small-en-v1.5 "Document 1" "Document 2"`
-
+### 1. Text Embedding Sample (`text_embeddings.py`)
+- **Description:**
+  Demonstrates inference of text embedding models using OpenVINO GenAI. Converts input text into vector embeddings for downstream tasks such as retrieval or semantic search.
+- **Run Command:**
+  ```sh
+  python text_embeddings.py <MODEL_DIR> "Document 1" "Document 2"
+  ```
 Refer to the [Supported Models](https://openvinotoolkit.github.io/openvino.genai/docs/supported-models/#text-embeddings-models) for more details.
 
+### 2. Text Rerank Sample (`text_rerank.py`)
```

Comment:
Extend the list of supported modesl

Reply:
I also need to add a full case description in docs. Let's do it in a separate PR. I created a separate ticket to track this.